connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/RestServer.java

```diff
@@ -306,6 +300,18 @@ public void initializeResources(Herder herder) {
         log.info("REST resources initialized; server is started and ready to handle requests");
     }
 
+    protected abstract Collection<ConnectResource> regularResources();
+
+    protected abstract Collection<ConnectResource> adminResources();
+
+    protected void configureRegularResources(ResourceConfig resourceConfig) {
+        // No-op by default
+    }
+
+    protected void configureAdminResources(ResourceConfig adminResourceConfig) {
```

Comment:
this method is not called or implemented

Reply:
Thanks--fixed. It's not necessary at the moment (no subclass will implement it) but it may be useful down the road and it's symmetrical with `configureRegularResources`, so I've added a couple call sites for it in `RestServer::initializeResources`. LMK if you think it'd be better to leave out entirely.